makefile.shade

```diff
@@ -309,7 +309,7 @@ var buildTarget = "compile"
             }
         }
 
-        Environment.SetEnvironmentVariable("NUGET_VOLATILE_FEED_ASPNETVNEXT", Path.Combine(coherencePath, "packages-expanded"));
+        Environment.SetEnvironmentVariable("NUGET_VOLATILE_FEED_AspNetCore", Path.Combine(coherencePath, "packages-expanded"));
```

Comment:
The casing matches what we have in NuGet.configs now - https://github.com/aspnet/Common/blob/dev/NuGet.config#L5.